subprojects/common/file-storage/src/main/kotlin/com/avito/filestorage/RemoteStorageFactory.kt

```diff
@@ -8,21 +8,24 @@ import java.util.concurrent.TimeUnit
 
 object RemoteStorageFactory {
 
-    private const val TIMEOUT_SEC = 120L
+    private const val TIMEOUT_SEC = 30L
+    private const val CONNECTION_TIMEOUT_SEC = 10L
 
     fun create(
         endpoint: String,
         httpClientProvider: HttpClientProvider,
         loggerFactory: LoggerFactory,
         timeProvider: TimeProvider,
-    ): RemoteStorage = HttpRemoteStorage(
-        endpoint = requireNotNull(endpoint.toHttpUrlOrNull()) { "Can't parse endpoint: $endpoint" },
-        httpClient = httpClientProvider.provide()
-            .connectTimeout(TIMEOUT_SEC, TimeUnit.SECONDS)
-            .writeTimeout(TIMEOUT_SEC, TimeUnit.SECONDS)
-            .readTimeout(TIMEOUT_SEC, TimeUnit.SECONDS)
-            .build(),
-        loggerFactory = loggerFactory,
-        timeProvider = timeProvider
-    )
+    ): RemoteStorage {
+        return HttpRemoteStorage(
+            endpoint = requireNotNull(endpoint.toHttpUrlOrNull()) { "Can't parse endpoint: $endpoint" },
+            httpClient = httpClientProvider.provide()
+                .connectTimeout(CONNECTION_TIMEOUT_SEC, TimeUnit.SECONDS)
```

Comment:
I looked through kibana warnings and for the most part, is about failures on device sending files.
I realize that we can't block 120 sec because we don't have critical attachments and wait them so long